test/features/pdf.js

```diff
@@ -9,10 +9,10 @@ describe('PDF Service', () => {
     before(() => server.start());
     after(() => server.stop());
 
-    // TODO: PDF tests a very unreliable, so skip them until T181084 is resolved
-    it.skip('Should get PDF for a page', () => {
+    it('Should get PDF for a page', () => {
         return preq.get({
-            uri: `${server.config.bucketURL('ru.wikipedia.org')}/pdf/%D0%94%D0%B0%D1%80%D1%82_%D0%92%D0%B5%D0%B9%D0%B4%D0%B5%D1%80`
+            uri: `${server.config.bucketURL('en.wikipedia.beta.wmflabs.org')}/pdf/%D0%94%D0%B0%D1%80%D1%82_%D0%92%D0%B5%D0%B9%D0%B4%D0%B5%D1%80`,
+            timeout: 30000
```

Comment:
Hm, no, instead of limiting the timeout here, the mocha timeout should be increased. Add `this.timeout(90000)` before the test definitions, i.e. before L12.

Reply:
Added the timeout as suggested.